core/network/address.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package network
+
+// Address describes a network address.
+type Address struct {
```

Comment:
I think this should also be removed from the network package and its usage replaced accordingly.

Reply:
Where do you recommend it be located in core?

Reply:
I mean having it there is fine, but it is *copied* from /juju/network/address rather than *moved*.
I think it should be deleted from juju/network/address and all usages updated to juju/core/network. That is what was done with the network types moved so far.

Now that I think about it, that is better for another patch, as it will pollute the intention here significantly. I'll approve to clear this one.

Reply:
thank you.  i'll make a card to do the move now.